torch/_dynamo/variables/user_defined.py

```diff
@@ -873,6 +873,31 @@ def odict_getitem(self, tx, key):
         )(collections.OrderedDict.__getitem__(self.value, key.as_python_constant()))
 
 
+class DebuggingVariable(UserDefinedObjectVariable):
```

Comment:
I think this wraps a function, so probably we should not extend from UDO. This variable can be python constant, but UDO can't.